js/script.js

```diff
@@ -0,0 +1,166 @@
+    ['ShiftLeft', 'KeyZ', 'KeyX', 'KeyC', 'KeyV', 'KeyB', 'KeyN', 'KeyM', 'Comma', 'Period', 'Slash', 'ArrowUp', 'ShiftRight'],
+    ['ControlLeft', 'MetaLeft', 'AltLeft', 'Space', 'AltRight', 'ControlRight', 'ArrowLeft', 'ArrowDown', 'ArrowRight']
+];
+const systemButton = ['Tab', 'CapsLock', 'ShiftLeft', 'ControlLeft', 'MetaLeft', 'AltLeft', 'Backspace', 'Del', 'Enter', 'ShiftRight', 'AltRight', 'ControlRight'];
+const specSymbol = ['~', '!', '@', '#', '$', '%', '^', '&', '*', '(', ')', '_', '+', '&#8592']
+let currentLang = 'eng';
+let lang = localStorage.getItem('lang');
+if(lang) {
+	currentLang = lang;
+} else {
+	localStorage.setItem('lang', currentLang);
+}
+let caps = false;
+
+const container = document.createElement('div');
+const textarea = document.createElement('textarea');
+const keyBoard = document.createElement('div');
+
+container.className = 'container';
+textarea.className = 'textarea';
+textarea.id = 'textarea';
+keyBoard.className = 'keyBoard';
+keyBoard.id = 'keyBoard';
+document.body.prepend(container);
+container.append(textarea);
+container.append(keyBoard);
+
+function createKeyBoard() {
+	if(document.getElementById('keyBoard')) {
+		keyBoard.innerHTML = ""; 
+	}
+	keyBoardIds.forEach((line, lineIndex) => {
+		const boardLine = document.createElement('div');
+		boardLine.className = 'boardLine';
+		line.forEach((button, buttonIndex) => {
+			const boardButton = document.createElement('div');
+			boardButton.className = 'boardButton';
+			boardButton.id = keyBoardIds[lineIndex][buttonIndex];
+			let buttonVal = language[currentLang][lineIndex][buttonIndex];
+			if(buttonVal.length === 1 && caps) {
+				buttonVal = buttonVal.toLocaleUpperCase();
+			}
+			boardButton.innerHTML = buttonVal;
+			boardButton.addEventListener("onmousedown", function() {
+				boardButton.classList.add('activeBtn');
+			});
+			boardButton.addEventListener("onmouseup", function() {
+				boardButton.classList.remove('activeBtn');
+			});
+			boardLine.append(boardButton);
+		})
+		keyBoard.append(boardLine);
+	})
+
+}
+
+
+
+
+function addText(id) {
+	let lineIndex;
+	let buttonIndex;
+	for(let i = 0; i < keyBoardIds.length; i++) {
+		if(keyBoardIds[i].indexOf(id) !== -1) {
+			lineIndex = i;
+			buttonIndex = keyBoardIds[i].indexOf(id);
+			break;
+		}
+	}
+	if(language[currentLang] && language[currentLang][lineIndex]) {
+		let newVal = language[currentLang][lineIndex][buttonIndex];
+		if(caps) {
+			newVal = newVal.toLocaleUpperCase();
+		}
+		textarea.value = textarea.value + newVal;
+	}
+	
+}
+function proceedSystem(id) {
+	if(id === 'CapsLock') {
+		caps = caps ? false : true;
+		createKeyBoard();
+	} else if(id === 'Tab') {
+		textarea.value = textarea.value + "\t";
+	} else if(id === "Enter") {
+		textarea.value = textarea.value + "\n";
+	}
+
+}
+
+document.addEventListener('keydown', function(event) {
+	if(event.altKey && event.ctrlKey) {
+		currentLang =  currentLang  === "eng" ?  "rus" : "eng";

```

Comment:
You are using thouse constants in a different place it is better to move it to some constants:
`const LANGUAGE_RUS = 'rus'`